js/main.js

```diff
@@ -0,0 +1,412 @@
+import keyboardKeys from './keyboardKeys.js';
+
+// Creating keyboard elements
+const body = document.querySelector('body');
+
+const divWrap = document.createElement('div');
+divWrap.className = 'wrapper';
+body.append(divWrap);
+
+const pageLangBtn = document.createElement('button');
+pageLangBtn.className = 'page-lang';
+pageLangBtn.insertAdjacentText('afterbegin', 'Lang');
+pageLangBtn.innerText = localStorage.getItem('virtualLang') || 'eng';
+divWrap.append(pageLangBtn);
+
+const textArea = document.createElement('textarea');
+textArea.className = 'textarea';
+textArea.id = 'textarea';
+textArea.setAttribute('type', 'textarea');
+divWrap.append(textArea);
+textArea.focus();
+
+
+const keyboard = document.createElement('div');
+keyboard.className = 'keyboard';
+keyboard.id = 'keyboard';
+divWrap.append(keyboard);
+
+// KeyboardLanguage changing
+if (localStorage.getItem('virtualLang') === null) {
+  localStorage.setItem('virtualLang', 'eng');
+}
+
+// Language chanching pressing on button on page
+function pageLangChanging() {
```

Comment:
rename **pageLangChanging** to **changeLanguage**
and simpify it. Verb should be the first word of function.  

```
function changeLanguage() {
  const newLanguage = localStorage.getItem(VIRTUAL_LANGUAGE_KEY) === EN ? RU : EN;
  localStorage.setItem(VIRTUAL_LANGUAGE_KEY, newLanguage);
  pageLangBtn.innerText = newLanguage.getItem(VIRTUAL_LANGUAGE_KEY);

  keyboard.querySelectorAll('.row').forEach(row => {
    row.querySelectorAll('.key').forEach(key => {
      const on = key.querySelector('.on');
      const off = key.querySelector('.off');
      on.classList.remove('on');
      on.classList.add('off');
      off.classList.remove('off');
      off.classList.add('on');
    });
  });
}

```